MapboxCoreNavigation/CoreConstants.swift

```diff
@@ -130,6 +132,13 @@ public extension Notification.Name {
      */
     static let routeControllerProgressDidChange: Notification.Name = .init(rawValue: "RouteControllerProgressDidChange")
     
+    /**
+     Posted when `RouteController` receives updated information about the current route.
+     
+     The user info dictionary contains the key `RouteController.NotificationUserInfoKey.routeProgressKey`.
+     */
+    static let routeControllerDidRefreshRoute: Notification.Name = .init(rawValue: "RouteControllerDidRefreshRoute")
```

Comment:
Is it weird that a notification about refreshing a _route_ comes with a route _progress_ object as user info? The idea is that the route progress object contains the route as well as other members that are affected by refreshing, so it’s handy to have immediate access to the whole route progress object.

Reply:
I think it's not really obvious, I'd expect route Route object to be there. We also have `routeControllerProgressDidChange`, but it doesn't contain only `RouteProgress` updates.